sdk/translation/azure-ai-translation-document/tests/test_model_updates.py

```diff
@@ -91,7 +71,16 @@ def test_document_translation_input_args(self, **kwargs):
     @DocumentTranslationPreparer()
     @DocumentTranslationClientPreparer()
     @recorded_by_proxy
-    def test_translation_target_args(self, **kwargs): 
+    def test_translation_target_args(self, **kwargs):
+        # Creating an instance using required positional arguments
+        target_positional = TranslationTarget(
+            target_url="https://t7d8641d8f25ec940prim.blob.core.windows.net/target-67890",
+            language="es"
+        )
```

Comment:
In this case, I'm looking for the values to be passed positionally, instead of by keyword. Like:

```python
target_positional = TranslationTarget(
    "https://t7d8641d8f25ec940prim.blob.core.windows.net/target-67890",
    "es"
)
```